source/CIPlanetNode.h

```diff
@@ -53,7 +62,9 @@ class PlanetNode : public cugl::scene2::AnimationNode {
     /** The nodes representing the layers of this planet */
     std::vector<LayerNode> _layerNodes;
   
-    void advanceFrame(LayerNode* node);
+//    void advanceFrame(LayerNode* node, bool isLockedIn, bool isLockingIn);
+    void advanceInnerLayerFrame(LayerNode* node);
+    void advanceOuterLayerFrame(LayerNode* node, bool isLockedIn, bool isLockingIn, int ii, int numLayers);
```

Comment:
Could you clean this up? I guess just remove the commented-out line and add some docstrings to the helper functions.